oom-frontend/src/app/shared/modules/collaborator/collaborator-editor/collaborator-editor.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533964)
```suggestion
import { ComponentFixture, TestBed } from "@angular/core/testing";
```